app/services/imports/case_logs_import_service.rb

```diff
@@ -226,6 +229,8 @@ def fields_not_present_in_softwire_data
     def check_status_completed(case_log, previous_status)
       if previous_status.include?("submitted") && case_log.status != "completed"
         @logger.warn "Case log #{case_log.id} is not completed"
+        @logger.warn "Case log with old id:#{case_log.old_id} is incomplete but status should be complete"
```

Comment:
Do we need the line above this? Could log old id and new id in the same warning?

Reply:
I just kept them separate but I could merge them into one warning if we want that 🙂